components/automate-chef-io/content/docs/backup.md

```diff
@@ -129,6 +129,10 @@ The command will show the backup progress for each service. If the backup is suc
 Success: Created backup 20180518010336
 ```
 
+## Storing Backups Offline for Airgapped Installations
```

Comment:
[nit] I'm ambivalent about "Offline", but willing to be convinced otherwise. As I read this section, it's really for people who want to create a single-file archive for each backup, which is probably (but not necessarily) to simplify putting the backup on storage that isn't attached to the Automate node. Is that accurate? If so, I'd probably call this section "Creating a single-file archive from a backup" or similar.

Reply:
Updated to match your feedback.